src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+@import "../variables";
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  background: #fff;
+  padding: 32px 16px 16px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    align-self: center;
+
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
```

Comment:
You can put value of this color in variable. So if this color will be changed, you need just rewrite variable value.